pkg/etcdenvvar/etcd_env.go

```diff
@@ -243,6 +245,26 @@ func getHardwareSpeedValues(envVarContext envVarContext) (map[string]string, err
 	return hwspeedhelpers.HardwareSpeedToEnvMap(speed)
 }
 
+func getEtcdDBSize(envVarContext envVarContext) (map[string]string, error) {
+	etcd, err := envVarContext.etcdLister.Get("cluster")
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve etcd CR: %v", err)
+	}
+
+	backendQuotaGiB := int64(8)
+	enabled, err := backendquotahelpers.IsBackendQuotaFeatureGateEnabled(envVarContext.featureGateAccessor)
+	if err != nil {
+		return nil, fmt.Errorf("failed check backend quota enabled: %w", err)
+	}
+	if enabled && etcd.Spec.BackendQuotaGiB != 0 {
```

Comment:
good thing that the testcase fails :) 

I think we need to also patch the feature gate into the release config like that:
https://github.com/openshift/release/commit/c978053a9dcc671e1ffbbf35a482df63c061b21e#diff-db0f271e35562d24e014bfd2f6c796daa79c27f26bd863aaaa303d59e6ce433bR134-R136

I'll send you a PR on slack

Reply:
gotcha 👍🏽 

But should not we turn on the FeatureGate for the `e2e test` in this PR only ? 

otherwise will stay on for coming PRs. no ?

Reply:
once the release config merges and is distributed to the build clusters, it should get picked up just fine

Reply:
i think we can override the featuregate on the cluster after being launched programmatically from within the test case, and then after we go GA we can just trim it, wdyt ?